src/test/java/htsjdk/tribble/index/tabix/TabixIndexTest.java

```diff
@@ -102,6 +114,8 @@ public void testQueryProvidedItemsAmount() throws IOException {
         ) // create TabixIndex straight from plaintext VCF
                 .write(plainTextVcfIndexFile); // write it
 
+        //TODO: you can pass in a .tbi file as the index for a plain .vcf, but if you *don't* pass in the file name and
```

Comment:
But is it correct to use a **.tbi** index with a plain text file? As you noted in a previous comment, this is not allowed by tabix... I suggest to blow up in this case!